app/views/posts/_allposts.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render partial: 'users/user', locals: { user: @user } %>
+
+<section id="posts-container">
+    <% @user.posts.each do |post| %>
+      <%= render partial: 'posts/post', locals: { post: post } %>
+
+      <% if post.comments.present? %>
+      <% post.comments.each do |comment| %>
+        <%= render partial: 'comments/commets', locals: { comment: comment } %>
+      <% end %>
```

Comment:
- [ ] It would be nice if you use the model method `recent_comments` instead of using the each loops 🤔 because there is not point of having these method if you don't utilize them and also they make your app efficient 👍